en/assembly/quick_start_pixracer.md

```diff
@@ -16,3 +16,16 @@
 ![Grau pixracer double](../../assets/flight_controller/pixracer/grau_pixracer_double.jpg)
 
 
+## Radio/Remote Control
+
+For general information about selecting a radio system, receiver compatibility, and binding see: [Remote Control Transmitters & Receivers](../getting_started/rc_transmitter_receiver.md).
+
+- PPM and S.BUS receivers must connect to the **RCIN** port.
+- PWM receivers (with individual cables for each channel) must connect

```

Comment:
Individual cables for each channel are also PPM reveivers.
PPM receivers with only one signal cable are PPM-Sum receivers.

Explanation:
The first sentence on http://ardupilot.org/copter/docs/common-ppm-encoder.html is basically wrong, the modulation method is exactly the same it's just that all the channels are sent one after the other when using PPM-Sum which is when they are timed correctly the sum of all the signals (hence the name).
The modulation is always PPM 1-2ms for one channel which is one specific sort of PWM modulation.
PWM only means you generally modulate the value by the duration of a pulse.
PPM is a specific type of PWM for RC and servo signals with the definition that 1ms pulse length is 0% and 2ms pulse length is 100%. The original definition says that there is a pulse every 20ms (50Hz) which allows the PPM-Sum signal that just fills the "dead time" with the pulses of all the other channels.

Reply:
Thanks for the explanation. I've updated that section so it looks like:

- PPM-Sum and S.BUS receivers must connect directly to the **RC** ground, power and signal pins (typically labeled RC or RCIN).
- PPM receivers that have an individual wire for each channel must connect to the RCIN channel *via* a PPM encoder [like this one](http://www.getfpv.com/radios/radio-accessories/holybro-ppm-encoder-module.html) (PPM-Sum receivers use a single signal wire for all channels).

I'll merge now - we can fix afterwards if I got that wrong!